climate-monitoring-client/src/main/java/climatemonitoring/CenterInfo.java

```diff
@@ -1,13 +1,71 @@
 package climatemonitoring;
 
+import climatemonitoring.core.Center;
+import climatemonitoring.core.ConnectionLostException;
+import climatemonitoring.core.DatabaseRequestException;
 import climatemonitoring.core.ViewState;
+import climatemonitoring.core.headless.Console;
+import climatemonitoring.core.utility.Command;
 
 class CenterInfo extends ViewState {
 
 	@Override
 	public void onHeadlessRender(String args) {
 
-		throw new UnsupportedOperationException("Unimplemented method 'onHeadlessRender'");
+		try {
+
+			Command c = new Command(args);
+			
+			String by = c.getCmd();
+			
+			if (by.equals("id")) {
+
+				String id = c.getArgs();
+				Center center = Handler.getProxyServer().getCenter(id);
+				
+				if (center != null) {
+
+					Console.write(center.getCenterID());
+					Console.write(center.getStreet());
+					Console.write(String.valueOf(center.getHouseNumber()));
+					Console.write(String.valueOf(center.getPostalCode()));
+					Console.write(String.valueOf(center.getCity()));
+					Console.write(center.getDistrict());
+				} else {
+
+					Console.write("Center not found.");
+				}
+			} else if (by.equals("index")) {
+
+				int index = Integer.parseInt(c.getArgs());
+				
+				Master m = (Master) Handler.getView().getState(ViewType.MASTER); 
+				Center[] centers = m.foundCenters; 
+				Center center = centers[index];
+				
+				if (center != null) {
+
+					Console.write(center.getCenterID());
+					Console.write(center.getStreet());
+					Console.write(String.valueOf(center.getHouseNumber()));
+					Console.write(String.valueOf(center.getPostalCode()));
+					Console.write(String.valueOf(center.getCity()));
+					Console.write(center.getDistrict());  
+				} else {
+					
+					Console.write("Center not found.");
+				}
+			} 
```

Comment:
You should also add an `else` statement when the user types something else other than "id" or "index". Something like this:
```
Console.write("Incorrect command syntax -->'" + c.getCmd() + "', expected [id, index]");
```